site/app/templates/course/SetFolderReleaseForm.twig

```diff
@@ -0,0 +1,98 @@
+{% extends 'generic/Popup.twig' %}
+{% block popup_id %}set-folder-release-form{% endblock %}
+{% block title %}Set Folder Release{% endblock %}
+{% block body %}
+
+    <center> <b>Choose a time to release the contents of this folder:</b>
```

Comment:
I feel "this folder" is ambiguous in the case that I clicked `Set All Released Dates`, because there may be various folders in the course material folder. It is hard to tell what `this folder` means.